core/src/main/scala/org/bitcoins/core/protocol/dlc/models/ContractInfo.scala

```diff
@@ -220,11 +220,7 @@ case class SingleContractInfo(
   }
 
   /** @inheritdoc */
-  override val max: Satoshis = contractDescriptor match {
-    case descriptor: EnumContractDescriptor =>
-      descriptor.values.maxBy(_.toLong)
-    case _: NumericContractDescriptor => totalCollateral
-  }
+  override val max: Satoshis = totalCollateral
```

Comment:
So now that i'm looking at the scaladoc for this, it appears that this might have been implemented correctly, but we used it in teh wrong place: 

https://github.com/Christewart/bitcoin-s-core/blob/a33e8b0ad0a3fcff1783dc7e9fd3be5d77891f2d/dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/DLCWallet.scala#L695

This should be

```scala
    val collateral = offer.contractInfo.totalCollateral - offer.collateral
```

thoughts @benthecarman @nkohen ?

Reply:
yeah i think that code above is correct and that line you linked needs to be fixed

Reply:
I can revert this change if thats the case and change `DLCWallet.scalaL695`

Reply:
should do a pass of places we call `contractInfo.max` and see if it should be totalCollateral instead

Reply:
Maybe rename `max` -> `maxOffererPayout` ?